src/vivarium_gates_mncnh/components/neonatal_causes.py

```diff
@@ -69,8 +69,8 @@ def get_normalized_csmr(self, index: pd.Index) -> pd.Series:
         normalized_csmr = raw_csmr * normalizing_constant
         # Account for structural zeros in preterm birth
         if self.neonatal_cause == NEONATAL_CAUSES.PRETERM_BIRTH:
-            ga_less_than_37 = pop[COLUMNS.GESTATIONAL_AGE] < 37
-            normalized_csmr.loc[ga_less_than_37] = 0
+            ga_greater_than_37 = pop[COLUMNS.GESTATIONAL_AGE] >= 37
```

Comment:
This is just reversing the logic. It probably should have been fixed in the other pr but branch management is getting a bit annoying to test with at this point.